include/GafferML/TensorToMesh.h

```diff
@@ -0,0 +1,87 @@
+//////////////////////////////////////////////////////////////////////////
+//
+//  Copyright (c) 2024, Lucien Fostier All rights reserved.
```

Comment:
Should there be a period after "Lucien Fostier"?

Reply:
Looks like this applies to a few other files too.

Reply:
addressed here ede42b00a2f4e181b1837cd5554a72f88e9cb0fa